packages/@tinacms/toolkit/src/packages/form-builder/FormBuilder.tsx

```diff
@@ -151,9 +173,23 @@ export const FormBuilder: FC<FormBuilderProps> = ({
         dirtySinceLastSubmit,
         hasValidationErrors,
       }) => {
+        const cannotSubmit =
```

Comment:
Small suggestion, positive boolean names's are a bit easier to read:
```
const canSubmit = 
!pristine && // ...
```

Reply:
Thanks for the suggestion. Resolved!

https://github.com/tinacms/tinacms/pull/3493/commits/41c88fe486dfe907c197ebc372ed41bf9132dab7